packages/common/src/account/kms/kmsKeyToAccount.ts

```diff
@@ -19,9 +19,9 @@ export type KmsAccount = LocalAccount<"aws-kms"> & {
  * @returns A Local Account.
  */
 export async function kmsKeyToAccount({
-  keyId,
+  keyId = process.env.AWS_KMS_KEY_ID || "",
```

Comment:
Defaulting in the account was inspired by this Foundry library https://github.com/foundry-rs/foundry/blob/master/crates/wallets/src/wallet.rs#L95-L97

Reply:
I did this because `keyId` is optional for the SDK but our `getKeyId` function returns a `string`. Gonna try something else